ci/script.sh

```diff
@@ -3,25 +3,25 @@ set -euxo pipefail
 main() {
     cargo check --target $TARGET
     if [ $TRAVIS_RUST_VERSION = nightly ]; then
-        cargo check --target $TARGET --features 'const-fn smaller-atomics'
+        cargo check --target $TARGET --features 'serde const-fn smaller-atomics'
```

Comment:
could you test `serde` outside this if block so it gets tested on stable? It's fine to just test the serde feature on stable since it doesn't depend on unstable features or interact with the outer features (they depend on stable).